express-graphql/dist/renderGraphiQL.js

```diff
@@ -64,7 +64,7 @@ const introductionDemoQuery = `# Welcome to 1BahnQL
 
 # More complex query examples:
 #
-# Querys 5 stations in a distance of 2000m with their corresponding facilities and location
+# Queries 5 stations in a distance of 2000m with their corresponding facilities and location
```

Comment:
Queries 5 stations **within a distance** of 2000m **and their** corresponding facilities and location